### ISSUE TITLE
Provide the final shell script as part of distribution

### ISSUE THREAD

**Issue by alexeagle** [Tinder/bazel-diff#30]
Right now users copy-paste the `bazel-diff-example.sh` to stitch together the bazel-diff commands. This means users end up modifying and having diverging copies, not able to upstream their improvements to share with other users.

Ideally some "shrink-wrapped" distribution would include the top-level entry point (maybe still using Bash).

**Comment by tinder-maxwellelliott** [comment]
> This means users end up modifying and having diverging copies, not able to upstream their improvements to share with other users.

I agree that ideally we could share one shell script to rule them all, the blocker here is that I do not know the exact steps required to build any Bazel project. For example, if we have revision A and revision B there may be a core tooling change that happens in B, that may require the developer to run some kind of setup script before we can run bazel-diff on revision B (think installing some command line tool). 

I worry that if we prescribe one approach then users who have some tooling that is not fully specified by Bazel will be left behind
